src/coreclr/System.Private.CoreLib/src/System/Attribute.CoreCLR.cs

```diff
@@ -436,6 +436,11 @@ private static AttributeUsageAttribute InternalGetAttributeUsage(Type type)
 
         private static Attribute[] CreateAttributeArrayHelper(Type elementType, int elementCount) =>
             elementType.ContainsGenericParameters ? new Attribute[elementCount] : (Attribute[])Array.CreateInstance(elementType, elementCount);
+
+        private static AmbiguousMatchException GetAmbiguousMatchException(Attribute attribute)
+        {
+            return new AmbiguousMatchException(SR.Format(SR.RFLCT_AmbigCust_Detailed, attribute.GetType().ToString(), attribute.ToString()));
+        }
```

Comment:
NIT:
```suggestion
        private static AmbiguousMatchException GetAmbiguousMatchException(Attribute attribute) =>
            new AmbiguousMatchException(SR.Format(SR.RFLCT_AmbigCust_Detailed, attribute.GetType().ToString(), attribute.ToString()));
```

Reply:
Actually why passing a same info twice? 

`attribute.GetType().ToString(), attribute.ToString()` produces `System.SerializableAttribute System.SerializableAttribute`

Reply:
Applied this fix and swapped in `SR.RFLCT_AmbigCust_Simple` for `SR.RFLCT_AmbigCust_Detailed`

Reply:
(and then simplified even more with rest of feedback)